pages/api/v4/[network]/metadata/token.js

```diff
@@ -38,6 +39,18 @@ const api = async (req, res) => {
         break;
     }
 
+    if (req.method === "POST") {
+      const body = JSON.parse(JSON.stringify(req.body));
```

Comment:
Seems unnecessary, why do you need to stringily and than parse?

Reply:
I wonder what is the difference... maybe because we never used the body until now, maybe there is a nextjs settings to parse body as json.